### PR TITLE
Introducing `acmcsuf.com`'s _first contributions_ initiative!

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Screenshot: 
image

### CODE REVIEW

File: src/routes/1st/index.svelte [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+    <h2 class="headers">Make necessary changes and commit them</h2>
+
+    <p class="brand-em">Want to make it rain?</p>
+
+    <br />
+
+    <p>
+      Step 1: Add your SVG or PNG in <code>/static/assets/1st/your-new-thing.(svg|png)</code> (See:
+      <a href="https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Basic_Shapes"
+        >Basic shapes
```
Comment (jaasonw) [inline src/routes/1st/index.svelte]: image
extra space here

File: src/routes/1st/index.svelte [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+  <section id="welcome">
+    <h2 class="headers">Welcome to ACM!</h2>
+
+    <p>
+      acmCSUF provides a social online platform, Discord, that brings passionate Titans together to
+      engage in multiple conversations regarding our community, the tech field, and many more. Join
+      <a href="/discord">our Discord server</a> to learn more about future events, meet new people, and
+      get involved with the community!
+    </p>
+  </section>
```
Comment (jaasonw) [inline src/routes/1st/index.svelte:75]: i dont really like the mix of center align and left align + the width of these containers is not consistent

File: src/routes/1st/index.svelte [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+</CommonHero>
+
+<Spacing --med="64px" />
+
+<main>
+  <section id="welcome">
+    <h2 class="headers">Welcome to ACM!</h2>
+
+    <p>
+      acmCSUF provides a social online platform, Discord, that brings passionate Titans together to
```
Comment (jaasonw) [inline src/routes/1st/index.svelte]: acmCSUF -> ACM at CSUF ?

File: src/routes/1st/index.svelte [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+      get involved with the community!
+    </p>
+  </section>
+
+  <section id="fork">
+    <h2 class="headers">Fork <code>acmcsuf.com</code></h2>
+
+    <p dir="auto">
+      <img
+        width="300"
```
Comment (jaasonw) [inline src/routes/1st/index.svelte:82]: images could be bigger i think

File: src/routes/1st/index.svelte [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+        </li>
+      </ul>
+    </details>
+  </section>
+
+  <section id="review">
+    <h2 class="headers">Submit your changes for review</h2>
+
+    <p>
+      Please feel free to contact us if you have any questions about acmCSUF&rsquo;s Privacy Policy
```
Comment (jaasonw) [inline src/routes/1st/index.svelte]: ACM at CSUF

File: src/routes/1st/index.svelte [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+      <img
+        src="https://camo.githubusercontent.com/71401ba5551a64aeac3838825a52ce7a7597cd8b54a0d7200d9454e2cbfbb13f/68747470733a2f2f6669727374636f6e747269627574696f6e732e6769746875622e696f2f6173736574732f526561646d652f7375626d69742d70756c6c2d726571756573742e706e67"
+        alt="submit pull request"
+        data-canonical-src="https://firstcontributions.github.io/assets/Readme/submit-pull-request.png"
+        style="max-width: 100%;"
+      />
+    </p>
+
+    <p dir="auto">
+      Soon I'll be merging all your changes into the master branch of this project. You will get a
```
Comment (jaasonw) [inline src/routes/1st/index.svelte]: remove first person reference and master branch should be main branch

File: src/routes/1st/index.svelte [hunk 7]
```diff
@@ -0,0 +1,322 @@
...
+        src="https://camo.githubusercontent.com/1c0cf8056422ff414eee75142b213c5970e085c2e33c0a6d69dc2639d98216f1/68747470733a2f2f6669727374636f6e747269627574696f6e732e6769746875622e696f2f6173736574732f526561646d652f636f70792d746f2d636c6970626f6172642e706e67"
+        alt="copy URL to clipboard"
+        data-canonical-src="https://firstcontributions.github.io/assets/Readme/copy-to-clipboard.png"
+        style="max-width: 100%;"
+      />
+    </p>
+
+    <p>For example:</p>
+    <pre><code>
+git clone https://github.com/this-is-you/acmcsuf.com.git
```
Comment (jaasonw) [inline src/routes/1st/index.svelte:134]: not mobile responsive as it does not word wrap or overflow: scroll

File: src/lib/1st/confetti/toggle-confetti.svelte [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+      return;
+    }
+
+    active = false;
+    await tick();
+    active = true;
+  }
+</script>
+
+<span on:click={click} class:relative>
```
Comment (jaasonw) [inline src/lib/1st/confetti/toggle-confetti.svelte:14]: not sure if unintelligent but cannot find this button to toggle the confetti

File: src/routes/1st/index.svelte (hunk 7 above)
Comment (jaasonw) [reply src/routes/1st/index.svelte:134]: https://user-images.githubusercontent.com/8981287/189439801-1b1f9276-63da-4234-a9ee-2a1ca5d1ef31.mp4

**Review by jaasonw — CHANGES_REQUESTED** [review]

File: src/routes/1st/index.svelte (hunk 2 above)
Comment (jaasonw) [reply src/routes/1st/index.svelte:75]: image

File: src/lib/1st/confetti/toggle-confetti.svelte (hunk 8 above)
Comment (EthanThatOneKid) [reply src/lib/1st/confetti/toggle-confetti.svelte:14]: I want the whole screen to toggle it.

**Review by jaasonw — APPROVED** [review]
did not look but lgtm cus urgency level 5

**Comment by diamondburned** [comment]
I think we should either make the confetti go out after a few seconds OR have it only visible when scrolled to the top. As it is, it makes reading the page almost impossible without manually toggling the confetti off. Visual flares shouldn't behave like this.

**Pull request merged**